eo-maven-plugin/src/main/resources/org/eolang/maven/add_rust/add_rust.xsl

```diff
@@ -37,6 +37,17 @@ SOFTWARE.
               <xsl:attribute name="loc">
                 <xsl:value-of select="../attribute(loc)"/>
               </xsl:attribute>
+              <dependencies>
+                <xsl:for-each select="../o/o">
+                  <xsl:if test="../attribute(base) = 'org.eolang.tuple' and attribute(base) = 'org.eolang.string'">
```

Comment:
@levBagryansky Same here. I believe we can replace `..` by `parent::@base`. Or something similar to this approach